src/components/react/IconList/IconWrapper.module.css

```
@@ -0,0 +1,41 @@
+.wrapper {
+    position: relative;
+}
+
+.icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    aspect-ratio: 1 / 0.96;
+    font-size: 36px;
+    color: var(--color-primary);
+    border: 1px solid var(--color-border);
+    border-radius: var(--radius-lg);
+}
+
+.name {
+    padding: 0;
+    margin: 0;
+    margin-top: var(--size-spacing-xxs);
+    font-size: var(--text-body-sm-size);
+    font-weight: bold;
+    line-height: var(--text-body-sm-line);
+    color: var(--color-text-sub)
```

Looks like I'd better centralize the text.
```suggestion
    color: var(--color-text-sub);
    text-align: center;
```